sentry-android-core/src/main/java/io/sentry/android/core/ActivityLifecycleIntegration.java

```diff
@@ -167,7 +169,10 @@ private void startTracing(final @NotNull Activity activity) {
                 (Date) null,
                 true,
                 (finishingTransaction) -> {
-                  activityFramesTracker.setMetrics(activity, finishingTransaction.getEventId());
+                  @Nullable Activity unwrappedActivity = weakActivity.get();
+                  if (unwrappedActivity != null) {
+                    activityFramesTracker.setMetrics(activity, finishingTransaction.getEventId());
+                  }
```

Comment:
I'd log in to the `else` branch, so people know why frames were not attached in this case.

Reply:
Updated

Reply:
![image](https://user-images.githubusercontent.com/10974305/182626453-ce067285-58b7-497d-9692-d1a47de04fff.png)

is still use activity , replace by unwrappedActivity?

Reply:
I think you looked at some old version of the code in this PR. It's using the `unwrappedActivity` now. I unintentionally commited the code I played around with to confirm the fix and then fixed with another commit.